### DISCUSSION TITLE
How do I reference CSS files from layers that are imported with Git?

### DISCUSSION THREAD

**Discussion by saschafuchs** [nuxt/nuxt#28115] [Questions]
Currently, I am developing multiple applications and importing layers stored in Git repositories.

Is there a way to import the CSS files located in the assets directory of these layers into a single CSS file? For instance, `@import '~layer/assets/css/uiStyle.css'` does not work, nor does `~layer/REPONAME/assets/css/uiStyle.css`

The current problem is that I have to manually copy all CSS files from the layers into the final repository so that Tailwind/PostCSS can generate the main CSS file (since the custom CSS files often use `@apply`). Instead of copying, it would be easier if I could reference the other layers directly. Is there a workaround for this? Everything I’ve found so far does not address layers imported via GitHub/GitLab.

**Comment by remihuigen** [comment]
@saschafuchs Did you find a solution or workaround? 

I'm facing similar issues, where I want to extend / override the CSS of my base layer (which includes Nuxt UI and tailwind). But as soon as I add a css file into the target layer, the base layer css file is ignored. Also can't import the original css
